libs/domain/user/address-list/src/address-list.component.ts

```diff
@@ -46,9 +49,17 @@ export class AddressListComponent extends ComponentMixin<AddressListItemOptions>
     distinctUntilChanged()
```

Comment:
Why do we need `distinctUntilChanged` here?

Reply:
Not necessary, dropped